quix-frontend/client/src/lib/runner/services/autocomplete/high_light_and_score.ts

```diff
@@ -0,0 +1,79 @@
+import {
+  createMatchMask,
+  findAllIndexOf,
+} from './autocomplete-utils';
+import { findRelevantPartOfPrefix } from "../../../sql-autocomplete/adapter/sql-autocomplete-adapter-utills";
+import { ICompleterItem as AceCompletion } from '../../../code-editor/services/code-editor-completer';
+import { QueryContext } from '../../../sql-autocomplete/sql-context-evaluator';
+
+
+export function enrichCompletionItemAfterDotObject(all: AceCompletion[]): AceCompletion[] {
```

Comment:
i think enrichCompletionItemAfterDotObject describes better the situation to the next user of the code.
because it refers to a specific scenario.

if you insist i'm willing to compromise on` enrichCompletionItemObjectChild` .